src/sentry/templates/sentry/layout.html

```diff
@@ -56,36 +55,6 @@
         {% show_system_status %}
     {% endif %}
 
-    {% block global_sidebar %}
-      {% if request.user.is_authenticated %}
-        <div id="blk_sidebar"></div>
-        <script>
-        $(function(){
-          var SidebarWrapper = createReactClass({
```

Comment:
Ooh I think we can get rid of https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/sidebar/index.jsx#L504 now too